javascript/javascript1/week3/seriesDurationOfMyLife.js

```diff
@@ -0,0 +1,23 @@
+const seriesDurations = [
+    { title: "Game of thrones", days: 5, hours: 14, minutes: 0 },
+    { title: "Prison Break", days: 2, hours: 18, minutes: 0 },
+    { title: "Gomorrah", days: 1, hours: 23, minutes: 0 },
+  ];
+  
+  const averageLifeSpanInMinutes = 80 * 365 * 24 * 60;
```

Comment:
Another nice and readable implementation; awesome! 🤩 There is some issue with the formatting here where all lines except the first seem to be indented weirdly.